libs/api/domains/driving-license-book/src/models/drivingSchool.response.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { DrivingSchoolType } from './drivingLicenseBookSchoolType.response'
+@ObjectType()
+export class DrivingSchool {
+  @Field()
```

Comment:
Is it fine to omit the decorator contents? F.x: `@Field(() => String)`

Reply:
Yes for the basic types like number and string it's fine.

Reply:
A bit confusing name ... first thought this was the actual school and thus it didn't make sense to attach a nationalId to the object. But this is more like the school that the person took their driving license, right?

But here, the nationalId needs to be marked as the ID for GraphQL.

Also be mindful of the project's prefix for GraphQL entities. This should be prefixed with DrivingLicenseBookDrivingSchool (if this was named DrivingLicenseBook, then the prefix would be redundant).

Not sure how to improve the name though (in order to better explain what this entity relates to).

Reply:
I changed the name to drivingLicenseBookSchool. 
This object represents the school that the user is an employee for. So the nationalId here is the schools national id. Im not sure what could be a better name here, the driving schools a student takes are referred to as tests and the schools are the organisations that provide those tests. The user for this application is an employee from the school.
I think this name is pretty representative but I'm very deep in the sauce so all other ideas appreciated!